data/nitishkumar07.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Nitishkumar J G",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student @ RNSIT",
+  "avatar": "https://github.com/nitishkumar07.png",
+  "links": [
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://www.linkedin.com/in/nitishkumarjg/",
+      "icon": "faLinkedin"
```

Comment:
The icon needs to be `FaLinkedin`, you can check available icons here https://linkfree.eddiehub.io/icons in the docs